### ISSUE TITLE
sidfile.py create_header() crashes if only one station is configured

### ISSUE THREAD

**Issue by fenrog** [sberl/supersid#24]
precondition in supersid.cfg:
```
number_of_stations = 1

[STATION_1]
call_sign = RJH
color = r
frequency = 20500
```

different options how to reproduce:
1) configure `viewer = tk`
- python3 supersid.py -c=../Config/supersid.cfg
- then select either one of 
    - File --> Save Raw Buffers
    - File --> Save Filtered Buffers
    - File --> Save Extended raw Buffers
    - File --> Save filtered as ...

2) configure `viewer = wx`
- python3 supersid.py -c=../Config/supersid.cfg
- then select either one of
    - File --> Save Raw Buffers
    - File --> Save Filtered Buffers
    - Plot --> Plot

3) configure `viewer = text`
- python3 supersid.py -c=../Config/supersid.cfg
- then select either one of
    - F) safe Filtered buffers
    - R) save Raw buffers
    - E) save Extended raw buffers

**Comment by fenrog** [comment]
This can be solved as
```
diff --git a/supersid/sidfile.py b/supersid/sidfile.py
index 9274096..aee5da6 100644
--- a/supersid/sidfile.py
+++ b/supersid/sidfile.py
@@ -361,8 +361,13 @@ class SidFile():
                                        if 'monitor_id' in self.sid_params
                                        else self.sid_params['monitorid'])
         if isSuperSid:
-            hdr += "# Stations = %s\n" % self.sid_params['stations']
-            hdr += "# Frequencies = %s\n" % self.sid_params['frequencies']
+            # if only one station is configured, self.sid_params['stations'] is not present
+            if 'stations' in self.sid_params:
+                hdr += "# Stations = %s\n" % self.sid_params['stations']
+                hdr += "# Frequencies = %s\n" % self.sid_params['frequencies']
+            else:
+                hdr += "# Stations = %s\n" % self.sid_params['stationid']
+                hdr += "# Frequencies = %s\n" % self.sid_params['frequency']
         else:
             hdr += "# StationID = %s\n" % self.sid_params['stationid']
             hdr += "# Frequency = %s\n" % self.sid_params['frequency']
```

**Comment by fenrog** [comment]
fixed with a7078a86c885b9a95f707c3b9c92310c7a52ce93
